gapic/utils/lines.py

```diff
@@ -115,7 +123,9 @@ def wrap(text: str, width: int, *, offset: Optional[int] = None, indent: int = 0
         text='\n'.join([textwrap.fill(
             break_long_words=False,
             initial_indent=' ' * indent,
-            subsequent_indent=' ' * indent,
+            # ensure that subsequent lines for lists are indented 2 spaces
+            subsequent_indent=' ' * indent + \
```

Comment:
Maybe add spaces around `=`.

Reply:
I tried making this change but the style check failed
```
(py39) partheniou@partheniou-vm-3:~/git/gapic-generator-python$ find gapic tests -name "*.py" -not -path 'tests/**/goldens/*' | xargs autopep8 --diff --exit-code
--- original/gapic/utils/lines.py
+++ fixed/gapic/utils/lines.py
@@ -124,7 +124,7 @@
             break_long_words=False,
             initial_indent=' ' * indent,
             # ensure that subsequent lines for lists are indented 2 spaces
-            subsequent_indent = ' ' * indent + \
+            subsequent_indent=' ' * indent + \
             ('  ' if token.strip().startswith('-') else ''),
             text=token,
             width=width,
```

Reply:
I wonder if enclosing the right side in parentheses, or breaking this down into if-else statements would work.